### PR TITLE
[Titles] Reduce H2 padding

### PR DESCRIPTION
**Pull request by jeremie-lucca**
## Description

H2 padding 8px > 6px

-----

image

-----

### CODE REVIEW

**Review by vvalentin-lucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2571/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13298/ui-diff)

File: packages/scss/src/commons/config.scss [hunk 1]
```diff
@@ -347,7 +347,7 @@ $sizes: (
 	XL: (
 		fontSize: 1.375rem,
 		lineHeight: 1.75rem,
-		verticalPadding: 0.5rem,
+		verticalPadding: 0.375rem,
```
Comment (AlexiaAlarza) [inline packages/scss/src/commons/config.scss:350]: why do we need to still add padding, instead of raise the lineHeight ?

Comment (vvalentin-lucca) [reply packages/scss/src/commons/config.scss:350]: For text that passes to the line.

**Pull request merged**